tests/wandb_agent_test.py

```diff
@@ -55,6 +58,7 @@ def train():
         run = wandb.init(entity="ign", project="ignored")
         sweep_projects.append(run.project)
         sweep_entities.append(run.entity)
+        run.finish()
```

Comment:
Same here, I would think the sweep agents are automatically calling `finish` when the function returns...